macstadium-staging/main.tf

```diff
@@ -0,0 +1,149 @@
+variable "index" {
+  default = 1
+}
+
+variable "travisci_net_external_zone_id" {
+  default = "Z2RI61YP4UWSIO"
+}
+
+variable "macstadium_vanilla_image" {
+  default = "travis-ci-ubuntu18.04-internal-vanilla-1525123339"
+}
+
+variable "jobs_network_subnet" {
+  default = "10.182.0.0/18"
+}
+
+variable "jobs_network_label" {
+  default = "Jobs-1"
+}
+
+variable "vsphere_janitor_version" {
+  default = "0a41b7f"
+}
+
+variable "vsphere_monitor_version" {
+  default = "0a459b3"
+}
+
+variable "collectd_vsphere_version" {
+  default = "e1b57fe"
+}
+
+variable "ssh_user" {
+  description = "your username on the wjb instances"
+}
+
+variable "threatstack_key" {}
+variable "librato_email" {}
+variable "librato_token" {}
+variable "fw_ip" {}
+variable "fw_snmp_community" {}
+variable "vsphere_user" {}
+variable "vsphere_password" {}
+variable "vsphere_server" {}
+variable "vsphere_ip" {}
+
+terraform {
+  backend "s3" {
+    bucket         = "travis-terraform-state"
+    key            = "terraform-config/macstadium-staging-hopethisworks-terraform.tfstate"
```

Comment:
We can probably take out the "hopethisworks" from here. I think it's a weird relic from the struggles of bringing up the MacStadium terraform the first time.